modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -380,6 +381,24 @@ enum { VIDEOWRITER_PROP_QUALITY = 1,    // Quality (0..100%) of the videostream
        VIDEOWRITER_PROP_FRAMEBYTES = 2, // (Read-only): Size of just encoded video frame
      };
 
+// gPhoto2 properties, if propertyId is less than 0 then work on widget with that __additive inversed__ camera setting ID
+// Get IDs by using CAP_PROP_GPHOTO2_WIDGET_ENUMERATE.
+// @see CvCaptureCAM_GPHOTO2 for more info
+enum {
```

Comment:
A good practice is to align all the comments for each parameter below at the same identation. Could you apply this? Again beauty fix for clarification.